map/src/components/header.tsx

```diff
@@ -74,8 +81,8 @@ const Header = (props: Props) => {
         >
           <MdAdd className="icon icon-left" />
           <span>
-            Add information
-            <span className={CLS_SCREEN_LG_ONLY}>&nbsp;to this map</span>
+            {t(s => s.mdAdd1)}
+            <span className={CLS_SCREEN_LG_ONLY}>&nbsp;{t(s => s.mdAdd2)}</span>
```

Comment:
Rather than have the translations split like this, i think it makes more sense to provide both a short and long string, and then switch between the two (splitting like this may not be possible in all languages), and will be complicated for RTL languages for sure.